writer.go

```diff
@@ -59,6 +59,11 @@ func (w *Writer) Write(b []byte) (int, error) {
 	return w.writeAndRetry(w.priority, string(b))
 }
 
+// RawWrite sends a log message with a custom priority
+func (w *Writer) RawWrite(p Priority, b []byte) (int, error) {
```

Comment:
I've tried to be similar to Write method, for me it does not matter what kind of data I pass to the method. But []byte matches syslog message struct.
I'll rename the method.